views/layouts/admin_main.handlebars

```diff
@@ -164,16 +178,17 @@
     <!-- ============================================================== -->
     <!-- End Wrapper -->
     <!-- ============================================================== -->
-   
+
 </body>
 {{!-- JavaScript --}}
-  <script>
-        let date = "{{viewData.rhapsody.date}}";
-        let newDate = new Date(date);
-        let DD = newDate.getDate() < 10 ? '0' + newDate.getDate() : newDate.getDate();
-        let MM = newDate.getMonth() < 10 ? '0' + newDate.getMonth() : newDate.getMonth();
-        let YYYY = newDate.getFullYear();
-        let YYYYMMDD = YYYY + '-' + MM + '-' + DD;
-        document.getElementById("rhapsodyDate").value = YYYYMMDD;
-    </script>
+<script>
+    let date = "{{viewData.rhapsody.date}}";
```

Comment:
using es6 on the client side is often a bad idea, since all browsers don't yet support this, use var instead
```suggestion
    var date = "{{viewData.rhapsody.date}}";
```